lib/services/libvirtd.pm

```diff
@@ -0,0 +1,132 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Package for libvirtd service check tests for migration
+#
+# Maintainer: Wegao <wegao@suse.com>
+
+package services::libvirtd;
+use base 'opensusebasetest';
+use testapi;
+use utils;
+use strict;
+use warnings;
+
+#default guest vm params
+our %guest_params = (
+    'name'        => 'nested_vm_test',
+    'ram' => '',
```

Comment:
ram != memory? right?

Reply:
yes, ram is for ram parameter of virsh command